packages/flutter/lib/src/material/slider.dart

```diff
@@ -1002,8 +1015,12 @@ class _RenderSlider extends RenderBox {
 
     config.isSemanticBoundary = isInteractive;
     if (isInteractive) {
+      config.textDirection = textDirection;
       config.onIncrease = _increaseAction;
       config.onDecrease = _decreaseAction;
+      config.value = '${(value * 100).round().clamp(0, 100)}%';
```

Comment:
Hmm... When I configure a slider as follows:

```dart
new Slider(
  min: 200,
  max: 220,
  divisions: 10,
)
```

I would expect VoiceOver or TalkBack to read out 200, 210, or 220 as values and not a percent value indicating where between 200 and 220 I am (especially since it doesn't tell me what 0% and what 200% is(? I am imagining a slider to set the max. price I am willing to pay for a flight for example.

Reply:
The percentage behavior matches the default SeekBar behavior in Android (continuous or discrete).  Lets discuss tomorrow what the desired a11y behavior should be.

Reply:
Doing some more investigation - it is total possible to support values with a flag (normalizeSemanticsValue?) but because of how we lerp/unlerp doubles it is fairly easy to get a number like 20.000000000000001 which TalkBack will read out in its entirety.  We could always round to the nearest whole number and document that on the flag.

Thoughts @goderbauer ?